### PR TITLE
Add spinning sound to the spinner

### LINKED ISSUE #4631: Add back spinner "spinning" sound
**Describe the missing feature:** 
The sound effects for spinner is missing in osu!lazer even if you are using custom skin.

### PR DESCRIPTION
**Pull request by frenzibyte**
Plays a spinning sample in the background and the spinning rate would be calculated by `(spm / 180) * progress`, that's the nearest I got.

- [ ] Depends on ppy/osu-framework#2808
- [x] Depends on #5728 

Closes #4631

### CODE REVIEW

File: osu.Game.Rulesets.Osu.Tests/TestSceneSpinner.cs [hunk 1]
```diff
@@ -75,11 +75,7 @@ public TestDrawableSpinner(Spinner s, bool auto)
             protected override void CheckForResult(bool userTriggered, double timeOffset)
             {
                 if (auto && !userTriggered && Time.Current > Spinner.StartTime + Spinner.Duration / 2 && Progress < 1)
-                {
-                    // force completion only once to not break human interaction
-                    Disc.RotationAbsolute = Spinner.SpinsRequired * 360;
-                    auto = false;
-                }
+                    Disc.RotationAbsolute += (float)(Spinner.Duration / 120);
```
Comment (peppy) [inline osu.Game.Rulesets.Osu.Tests/TestSceneSpinner.cs]: where did this number come from?

Comment (iiSaLMaN) [reply osu.Game.Rulesets.Osu.Tests/TestSceneSpinner.cs]: just a random number to make the test spinner spin smoothly, you can change that if you have a better idea.

Comment (peppy) [reply osu.Game.Rulesets.Osu.Tests/TestSceneSpinner.cs]: no worries, just wanted to check.

**Comment by iiSaLMaN** [comment]
Seems like the spinning sample playback somehow regressed, it plays a very weird sound while spinning, will investigate the cause later. can be blocked for now.

**Pull request closed without merging**